src/api.rs

```diff
@@ -388,6 +388,34 @@ impl<L: Language> Iterator for SyntaxNodeChildren<L> {
     }
 }
 
+impl<L: Language> SyntaxNodeChildren<L> {
+    pub fn by_kind<'a>(
+        self,
+        matcher: &'a impl Fn(SyntaxKind) -> bool,
```

Comment:
```suggestion
        matcher: &'a dyn Fn(SyntaxKind) -> bool,
```
since this is internally stored as a trait object already we don't gain anything from this being generic right?

Reply:
I suppose not. I'll change this

Reply:
Changed